### ISSUE TITLE
Design System Components implementation

### ISSUE THREAD

**Issue by bdougie** [bdougie/open-sauced#337]
[opensauced.pdf](https://github.com/bdougie/open-sauced/files/3801162/opensauced.pdf)

- [x] contextcard
- [x] Hint card
- [x] Note card
- [x] Onboard card
- [x] repo data card
- [x] Button Card
- [x] Repo Details Card
- [x] Issues Card
- [x] Repo list card
- [x] Repo creation card

Issue Card, Context Card, Repo Details Card
<img width="848" alt="Screenshot 2019-11-02 19 18 17" src="https://user-images.githubusercontent.com/5713670/68079358-8c7dda00-fda5-11e9-8dc7-40e425ef98f3.png">

Repo List Cards
<img width="555" alt="Screenshot 2019-11-02 19 17 45" src="https://user-images.githubusercontent.com/5713670/68079368-943d7e80-fda5-11e9-8766-1083a8464004.png">

Context card
![Screenshot 2019-11-04 10 08 35](https://user-images.githubusercontent.com/5713670/68145625-14b9c780-feeb-11e9-997f-6b5539334de7.png)

TitleParagraphCard
![Screenshot 2019-11-04 10 08 10](https://user-images.githubusercontent.com/5713670/68145651-2307e380-feeb-11e9-8089-8b5d83d37784.png)

Onboarding card
![Screenshot 2019-11-04 10 08 29](https://user-images.githubusercontent.com/5713670/68145639-1b483f00-feeb-11e9-9788-c9a6335abee3.png)

Hint Card & Button Card
![Screenshot 2019-11-04 10 08 21](https://user-images.githubusercontent.com/5713670/68145741-50549180-feeb-11e9-8b4c-7d6310f3818f.png)

**Comment by bdougie** [comment]
[illustrations SVGs](https://www.dropbox.com/sh/x5pydklkce0tp5s/AADbrkGODzL7zcqlDawLwM__a?dl=0)

**Comment by ckopecky** [comment]
This is what I am working on! I will start at the top of the checklist and work my way down as I get things done.

**Comment by ckopecky** [comment]
@bdougie context card finished. Please let me know if I need to tweak it at all.

**Comment by ckopecky** [comment]
- [x] contextcard
 - [x]  Hint card
 - [x] Note card
 - [x] Onboard card
 repo data card
 - [x] Button Card
 - [x] Repo Details Card
 Issues Card
 - [x] Repo list card
 - [x] Repo creation card

@bdougie 
Update -- all code is pushed to my fork -- 

Will finish tomorrow

C

**Comment by bdougie** [comment]
I all forks merged, be sure to rebase master at this point to avoid merge conflicts. The majority of the tweaking to the components I can quickly run through when I implemented the components into the  UI. 

At this point there is only 1 component missing and it is the one I misnamed in the OP:

Will  you be able to address this? @ckopecky 

 Onboard card
![](https://user-images.githubusercontent.com/5713670/68145639-1b483f00-feeb-11e9-9788-c9a6335abee3.png)
